docs/support/account.md

```diff
@@ -0,0 +1,73 @@
+# Account Management
```

Comment:
Need to define the terms Public Officer and Vendor to be clear?

Reply:
don't really need. it's covered in quick start